scripts/write_notes_to_couchdb.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+DESC="""Module called by other EPP scripts to write notes to couchdb
+"""
+import json
+import yaml
+import couchdb
+import smtplib
+import os
+from email.mime.text import MIMEText
+
+
+def write_note_to_couch(pid, timestamp, note, lims):
+    configf = '~/.statusdb_cred.yaml'
+    with open(os.path.expanduser(configf)) as config_file:
+        config = yaml.safe_load(config_file)
+    if not config['statusdb']:
+        email_error('Statusdb credentials not found in {}\n '.format(lims), 'genomics-bioinfo@scilifelab.se')
+        email_error('Running note save for {} failed on LIMS! Please contact {} to resolve the issue!'.format(pid, 'genomics-bioinfo@scilifelab.se'), note['email'])
+        sys.exit(1)
+    url_string = 'http://{}:{}@{}:{}'.format(config['statusdb'].get('username'), config['statusdb'].get('password'),
+                                              config['statusdb'].get('url'), config['statusdb'].get('port'))
+    couch = couchdb.Server(url=url_string)
+    if not couch:
+        email_error('Connection failed from {} to {}'.format(lims, config['statusdb'].get('url')), 'genomics-bioinfo@scilifelab.se')
+        email_error('Running note save for {} failed on LIMS! Please contact {} to resolve the issue!'.format(pid, 'genomics-bioinfo@scilifelab.se'), note['email'])
+
+    proj_db = couch['projects']
+    doc_id = proj_db.view('project/project_id')[pid].rows[0].value
```

Comment:
Wonder if this works if the project is not found (as tested for below). I guess `rows[0]` would fail?